Formula/g/gauge.rb

```diff
@@ -2,7 +2,7 @@ class Gauge < Formula
   desc "Test automation tool that supports executable documentation"
   homepage "https://gauge.org"
   url "https://github.com/getgauge/gauge/archive/refs/tags/v1.6.13.tar.gz"
```

Comment:
CC @getgauge/core, the version hasn't been updated by the automated PR tooling.

Reply:
Thanks for letting us know. The sha seems correct for `1.6.14` implying the rest of the automation is working, so not sure why the automation didn't work in this case.

Can close this PR if you like (or if you have maintainer permissions, edit the URL to `1.6.14` if you feel like correcting ot).